### DISCUSSION TITLE
(how) Does SSG work in 1.0.0 alpha?

### DISCUSSION THREAD

**Discussion by fschoenfeldt** [alephjs/aleph.js#496] [Q&A] [answered]
I'm confused as the old versions ran via the `aleph build` command that built the static sites.

Right now in the alpha, I can't find any way to launch the SSG, halp 😄

**Comment by ije** [comment, marked as answer]
sorry, aleph 1.0 wouldn't provide the SSG feature, but will support cache the SSR output to DB/fs

**Reply by mandric** [reply]
Can you recommend an alternative to Aleph 1.0 for SSG?  Use an older version or a branch or another framework you can recommend?

**Reply by ije** [reply]
@mandric  in fact currently you can use the ssg feature, just did't update the docs...

**Reply by ije** [reply]
here is an example:

```ts
serve({
  plugins: [
    denoDeploy({ modules }),
    mdx({
      remarkPlugins: [remarkFrontmatter, remarkGFM],
      rehypePlugins: [rehypeHighlight, rehypeSlug],
      providerImportSource: "@mdx-js/react",
    }),
    react({ ssr: true }),
  ],
  build: { ssg: true }
});
```

and you can choose the routes to generate htmls: `ssg: { exclude: /^\/admin/ }`
